<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hintHelper</class>
 <widget class="QWidget" name="hintHelper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QListWidget" name="list">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0</string>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAdd_hint">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
   </property>
   <property name="text">
    <string>Add hint</string>
   </property>
  </action>
  <action name="actionRemove_Hint">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/close.png</normaloff>:/icons/close.png</iconset>
   </property>
   <property name="text">
    <string>Remove Hint</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionEdit_hint">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/edit.png</normaloff>:/icons/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit hint</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
